WordPress/playground-tools
# Update Playground dependencies to 0.6.7

## What?


* Update playground dependencies to 0.6.3
* Adapt wp-now code to use the modified APIs.
  * php.request uses body instead of formData
  * wp-login.php is required to login users through the blueprint step.


## Why?



To remove crypto dependency in node applications

## How?



Updating the main `package.json`

## Testing Instructions






- Run `nvm use && npm install`
- Run `npm run build`
- Run `node dist/packages/wp-now/cli.js start --blueprint=b.json` where b.json is a blueprint with 
- Observe the blueprint executes correctly without any error about `TypeError: crypto.randomUUID is not a function`

```
{
	"$schema": "https://playground.wordpress.net/blueprint-schema.json",
	"landingPage": "/wp-admin/plugins.php",
	"preferredVersions": {
		"php": "8.0",
		"wp": "latest"
	},
	"steps": [
		{
			"step": "login",
			"username": "admin",
			"password": "password"
		},
		{
			"step": "installPlugin",
			"pluginZipFile": {
				"resource": "wordpress.org/plugins",
				"slug": "notification"
			},
			"options": {
				"activate": true
			}
		}
	]
}
```
